### PR TITLE
Create 2020-01-21-200wad-logo.md

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2020-01-21-200wad-logo.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+status: publish
+categories: [Writing]
+image:
+image2:
+author: Valentino Urbano
+---
+
+Note: This post is about the new logo @ [200WaD][1]. If you're not using the website you might not be interested in it, feel free to skip it.
+
+I really liked the original 200 words a day logo (the old logo) even though baz said multiple times that he made it himself and didn't really take too much time with it. I consider that a plus since it came out really well despite that.
```
Comment (MsSlacker) [inline _posts/2020-01-21-200wad-logo.md]: What's the  boyfriend in () for? Or just repeats stuff... Unless it's a link?

Comment (MsSlacker) [reply _posts/2020-01-21-200wad-logo.md]: Spell people's names with big letters

File: _posts/2020-01-21-200wad-logo.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Simple, but with a meaning behind it. Obvious to its meaning, but not too much to cheapen it and make it feel too basic.
+
+This new one feels that way to me. I agree that a change was needed since the rebranding, but I'm not a huge fan of the current choice.[^1]
```
Comment (MsSlacker) [inline _posts/2020-01-21-200wad-logo.md]: also feels or doesn't feel, pick something

Comment (MsSlacker) [reply _posts/2020-01-21-200wad-logo.md]: since - > because? Following?

File: _posts/2020-01-21-200wad-logo.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Simple, but with a meaning behind it. Obvious to its meaning, but not too much to cheapen it and make it feel too basic.
+
+This new one feels that way to me. I agree that a change was needed since the rebranding, but I'm not a huge fan of the current choice.[^1] 
+
+I always found making good logos to be hard. A basic logo is really easy to make instead, but in most cases what you end up with is nothing more than "meh". It is ok and might even look good, but it doesn't really transmit what you want it to.
+
+To make a really good logo you need a lot of tries to really home in to the right design and that takes time.[^2] On the other hand, as with everything, perfectionists never ship anything if they don't tell themselves it is good enough. Time to put it out there.
+
+What is your process? Do you start with an idea and work on the logo from that idea? Do you take a basic shape and morph it to something more?
```
Comment (MsSlacker) [inline _posts/2020-01-21-200wad-logo.md]: What do you do?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2020-01-21-200wad-logo.md (hunk 1 above)
Comment (valeIT) [reply _posts/2020-01-21-200wad-logo.md]: > What's the boyfriend in () for? Or just repeats stuff... Unless it's a link?

Yeah wanted to add it, but could not find it. So will remove it for now

File: _posts/2020-01-21-200wad-logo.md (hunk 3 above)
Comment (valeIT) [reply _posts/2020-01-21-200wad-logo.md]: I'm terrible at logos so I don't have a process :P

File: _posts/2020-01-21-200wad-logo.md [hunk 4]
```diff
@@ -11,19 +11,19 @@ image2:
 author: Valentino Urbano
 ---
 
-Note: This post is about the new logo @ [200WaD][1]. If you're not using the website you might not be interested in it, feel free to skip it.
+Note: This post is about the new logo @ [200WaD][1]. If you're not using the website you might not be interested in this article, feel free to skip it.
 
-I really liked the original 200 words a day logo (the old logo) even though baz said multiple times that he made it himself and didn't really take too much time with it. I consider that a plus since it came out really well despite that.
+I really liked the original 200 words a day logoeven though Baz said multiple times that he made it himself and didn't really take too much time with it. I consider that a plus since it came out really well despite that.
```
Comment (MsSlacker) [inline _posts/2020-01-21-200wad-logo.md]: What is logoeven

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2020-01-21-200wad-logo.md (hunk 4 above)
Comment (valeIT) [reply _posts/2020-01-21-200wad-logo.md]: lol

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**